IoT-Sample/Swift/IoTSampleSwift/ConnectionViewController.swift

```diff
@@ -228,32 +294,66 @@ class ConnectionViewController: UIViewController, UITextViewDelegate {
         tabBarViewController.viewControllers = [ self, configurationViewController ]
         logTextView.resignFirstResponder()
 
-        // Initialize AWSMobileClient for authorization
-        AWSMobileClient.sharedInstance().initialize { (userState, error) in
-            guard error == nil else {
-                print("Failed to initialize AWSMobileClient. Error: \(error!.localizedDescription)")
-                return
-            }
-            print("AWSMobileClient initialized.")
-        }
+        //Initialize control plane
+        // Initialize the Amazon Cognito credentials provider
+        let credentialsProvider = AWSCognitoCredentialsProvider(regionType:AWS_REGION,
+                                                                identityPoolId:IDENTITY_POOL_ID)
+        let controlPlaneServiceConfiguration = AWSServiceConfiguration(region:AWS_REGION, credentialsProvider:credentialsProvider)
+
+        //IoT control plane seem to operate on iot.<region>.amazonaws.com
+        //Set the defaultServiceConfiguration so that when we call AWSIoTManager.default(), it will get picked up
+        AWSServiceManager.default().defaultServiceConfiguration = controlPlaneServiceConfiguration
+        iotManager = AWSIoTManager.default()
+        iot = AWSIoT.default()
+
```

Comment:
It would be useful to separate these different connection strategies into completely separate methods--maybe even completely separate files (although that'll be additional work to do things like updating the text view), to aid customers who copy/paste reference implementations. The more self-encapsulated we can make it, the easier it will be for someone to get up & running. That would also lay the groundwork for some future "HOWTO" that runs along the lines of "Copy these 2 files, link them into your Xcode project, invoke from your view controller, ... Profit!"

Reply:
Yeah.. it really ought to be in a different view controller, and potentially separate app altogether....

But.. for the control plane approach, we need to instantiate both control plane and data plane, and for data plane, we only need to instantiate the data plane....  Im tempted to commit this approach first, and then break this to two separate view controllers in a subsequent pr.